### PR TITLE
Update code-of-conduct.md

### PR DESCRIPTION
**Pull request by Smokyish**
My version for Code of Conduct.
Edited and updated version of @ethereumnick commit https://github.com/status-im/wiki.status.im/pull/11/commits/e53f48411394be31c02f2d20f6376952f9f46302

I think CoC should be short and informative, many parts of the previous CoC and commit would be better suited for Mission and Core Values or some other wiki item.

### CODE REVIEW

**Comment by ethereumnick** [comment]
Looks good. two things struck me early on though.

There is a behavioural psychology reason for stating something to the effect that  "This document is not a rulebook," and or " We expect community members to follow the spirit as much as the letter of this documentation".  The reason is that self monitoring is more effective when there is ambiguity over what the rules may be.  Ambiguity also gives people a way to save face and feel better about resolving disagreements with others and still feeling internally consistent. In other words, people don't like to get caught breaking rules or guidelines but they feel worse about reintegration after breaking rules than after 'misunderstandings'.  

"showing respect and humanity" : 
Humanity is a double edged sword.  As a species we are naturally capable of being vindictive, spiteful, violent and aggressive. I suspect Status.im would wish to exclude these **adaptive** behaviours from our community opting instead for earning and showing respect through the 'better angles of our nature'.

**Comment by Smokyish** [comment]
I understand your point about behavioural psychology, i would hope to have the wording so that it gives people information on what is expected of them and enough freedom to be creative and not to stifle innovation, which is why i don't see that it should be a strict set of rules rather than a set of guidelines.

The word `Humanity` in my opinion in this context refers to the virtue humanity, as in doing humanitarian acts towards others. Perhaps it could be rephrased to `showing respect and basic humane ethics and morals for one another`.

File: docs/getting-started/code-of-conduct.md [hunk 1]
```diff
@@ -1,95 +1,65 @@
...
+
+The Code of Conduct serves to establish guidelines on how to conduct your behaviour. We are all responsible for maintaining and developing this shared environment.  
+We want to have a common productive, collaborative and socially responsible space. We hope that all our community members share our views and nurture this spirit and culture in their dealings with the Status community.  
+
 
 ## Community
+Status is about showing respect and humanity for one another: the word itself captures the spirit of social esteem in a human community.
 
-Status is about showing humanity to one another: the word itself captures the spirit of being human.
+We want to grow productive and agile place that can welcome new ideas in a complex field. We wish to improve every process in every development cycle and foster collaboration between groups with very different needs, interests and skills.
```
Comment (0xc1c4da) [inline docs/getting-started/code-of-conduct.md]: > We want to grow productive  

wut?

File: docs/getting-started/code-of-conduct.md [hunk 2]
```diff
@@ -1,95 +1,65 @@
...
 
-We want a productive, happy and agile community that can welcome new ideas in a complex field, improve every process every year, and foster collaboration between groups with very different needs, interests and skills.
+We gain strength from diversity, and actively seek participation from those who enhance it. This Code of Conduct exists to ensure that diverse groups collaborate to mutual advantage and enjoyment.  
 
-We gain strength from diversity, and actively seek participation from those who enhance it. This code of conduct exists to ensure that diverse groups collaborate to mutual advantage and enjoyment. We will challenge prejudice that could jeopardise the participation of any person in the project.
```
Comment (0xc1c4da) [inline docs/getting-started/code-of-conduct.md]: > We will challenge prejudice that could jeopardise the participation of any person in the project.

why is this removed?

Comment (Smokyish) [reply docs/getting-started/code-of-conduct.md]: It's not removed, but moved under - **Be respectful**

**Comment by 0xc1c4da** [comment]
So, I guess my biggest question is what is the description of the pull request and what are the highlights? What does this PR do that we didn't have before? What is the goal here?

There are some minor edits I agree with but I'm not really sure with the reorganisation as a whole, how do we plan to maintain this document if it changes upstream - having a different order seems like it will create more work for us in maintaining this document

**Comment by Smokyish** [comment]
The ultimate goal was to do a re-write and have a CoC that is unique and customized to Status. It is still heavily based on the Ubuntu CoC, but after these changes it should be easier to maintain and add changes that work specifically for Status and not be as dependant on changes from Ubuntu.

**Comment by ethereumnick** [comment]
> We want to grow productive ...

works as

>We want to grow **a** productive and agile place that can welcome new ideas ...

but could be 'build', 'create', 'nurture' etc.

I agree that the Ubuntu CoC is a starting point rather than a code base.  We don't have to fear getting out of sink with up stream.

I think that in general large scale changes in git are really clunky and painful. perhaps there should be incremental commit policy for large documents like these?

I also think that there should be an over all distillation of the docs so that no single doc is too long.  I don't think that means that we should drop loads just that it should be neatly nested in sub sections that keep the pages clean.  This will give the casual reader a rapid digest but also offer detail in any area someone wants to dig deep on.

all in all I think we should go with tatu's changes and take bite sized edits from now on.

thought?

**Comment by Smokyish** [comment]
@jarradh Can we get some thoughts or reasoning as to why you chose not to make any changes to the CoC, or is this still under review?

**Comment by 0xc1c4da** [comment]
Oh I chose the wrong one? There was a conflict in this PR so it couldn't be merged, the conflict was **THE ENTIRE DOCUMENT** and I just chose one of them.

Feel free to submit your changes and make sure the PR does not have any conflicts.

**Comment by 0xc1c4da** [comment]
I made a review you didn't change anything in this, why didn't you update the PR? and this has been merged with the conflict 'fixed'.

**Comment by Smokyish** [comment]
As Nick asked for thoughts on the matter, i didn't make any changes until there would be some agreement on what will be done for the CoC.

I'll make those changes from the review.

**Comment by 0xc1c4da** [comment]
Okay How about I just revert this merge because I did accept @ethereumnick 's changes in his pull request.
